tests/attr/test_kernel_shap.py

```diff
@@ -197,6 +197,7 @@ def test_multi_input_batch_kernel_shap(self) -> None:
             expected,
             additional_input=(1,),
             feature_mask=(mask1, mask2, mask3),
+            n_perturb_samples=300,
```

Comment:
nit: Why did we set, `n_perturb_samples` here ?

Reply:
This is increased from the default to ensure convergence within the necessary delta, random behavior is slightly different with the new sampling, which likely caused this.